RELEASING-AMAZON.md

```diff
@@ -0,0 +1,11 @@
+1. Start a branch `release/amazon-x.x.x`
+1. Run `flutter format .`
+1. Update CHANGELOG-LATEST.md with the changes for the current version (to be used by Fastlane for the github release notes)
+1. Run `bundle exec fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in pubspec.yaml, purchases_flutter.podspec and android/build.gradle.
```

Comment:
did you run it for this PR and it worked fine?

Reply:
yep! it only updates a handful of files, I didn't run into any issues